src/main/java/edu/harvard/iq/dataverse/search/SolrSearchServiceBean.java

```diff
@@ -82,70 +82,16 @@ public class SearchServiceBean {
     @Inject
     ThumbnailServiceWrapper thumbnailServiceWrapper;
     
-    /**
-     * Import note: "onlyDatatRelatedToMe" relies on filterQueries for providing
-     * access to Private Data for the correct user
-     *
-     * In other words "onlyDatatRelatedToMe", negates other filter Queries
-     * related to permissions
-     *
-     *
-     * @param dataverseRequest
-     * @param dataverses
-     * @param query
-     * @param filterQueries
-     * @param sortField
-     * @param sortOrder
-     * @param paginationStart
-     * @param onlyDatatRelatedToMe
-     * @param numResultsPerPage
-     * @return
-     * @throws SearchException
-     */
-    public SolrQueryResponse search(DataverseRequest dataverseRequest, List<Dataverse> dataverses, String query, List<String> filterQueries, String sortField, String sortOrder, int paginationStart, boolean onlyDatatRelatedToMe, int numResultsPerPage) throws SearchException {
-        return search(dataverseRequest, dataverses, query, filterQueries, sortField, sortOrder, paginationStart, onlyDatatRelatedToMe, numResultsPerPage, true, null, null);
+    
+    @Override
+    public String getServiceName() {
+        return "solr";
```

Comment:
Should we use the new SearchServiceFactory.INTERNAL_SOLR_SERVICE_NAME variable instead of hard-coding "solr" here? We'd have to make it public.